### DISCUSSION TITLE
Question about Nextcloud bookmark vs Nextcloud webdav

### DISCUSSION THREAD

**Discussion by sn2020aug** [floccusaddon/floccus#1589] [General]
I recently found floccus and is using it with Google Drive but have a question about Nextcloud. If I have access to a Nextclod installation, why do I want to use Nextcloud Bookmark and not Nextcloud via WebDAV?

**Comment by marcelklehr** [comment]
Hi, this is a great question actually. I'll put it in the FAQ :)

So, the advantage of Nextcloud Bookmarks is that you get a nice user interface in nextcloud allowing you to search and filter and manage your bookmarks in Nextcloud. The advantage of webDAV is that it's usually faster with less than ~10000 bookmarks (at some point downloading the bookmarks file everytime takes a while if you have a LOT of bookmarks).
In the future, the Floccus Android and ios Apps will also allow editing the notes and tags of bookmarks when syncing with Nextcloud Bookmarks, which won't work with pure WebDAV.

I hope this answers your question :)

**Reply by sn2020aug** [reply]
Would the issue of not being able to edit notes and tags be the same for Google Drive or git?

**Reply by marcelklehr** [reply]
Likely yes, so far
